components/prism-core.js

```diff
@@ -415,6 +420,8 @@ var script = document.currentScript || [].slice.call(document.getElementsByTagNa
 
 if (script) {
 	_.filename = script.src;
+	_.tags = (script.getAttribute('data-tags') || 'code').split(/\s+/g)
+				.filter(function(tag) {return !!tag;});
```

Comment:
To filter empty strings. For example the following code
```js
 '   code kbd   '.split(/\s+/g);
```
returns this array:
```js
[ '', 'code', 'kbd', '' ]
```
after the filter I get:
```js
[ 'code', 'kbd' ]
```